app/views/staffcos/_new.html.haml

```diff
@@ -0,0 +1,16 @@
+- content_for :modal_content do
+    .card
+    .inner-card
+        .card-title
+            %h1 Staffing Companies
+        #list-item.flex
+        = form_with scope: :staffco, url: staffcos_path, local: true, id: :create_sc do |form|
```

Comment:
You probably want to indent the form to be within `#list-item.flex` But, on a second thought, why do you need the id `list-item`?